### DISCUSSION TITLE
Unable to prove online learning for campus location

### DISCUSSION THREAD

**Discussion by WhyNineKay** [community/community#135469] [GitHub Education]
### Hi everyone,

Hi.

I am having huge amounts of trouble with the absolute abysmal reapplication process for the student developer pack.

My school is online based. Of course, my location is at my house, not at the school. Which I indicated with the checkbox: 
![image](https://github.com/user-attachments/assets/3f932a1b-c46f-4979-9771-c48088a4f9bc)

This halfbaked process then makes me "Upload an image supporting your reason for being far from campus." What am I supposed to take a picture of? An image of my schools website stating "it's online"? Not good enough. An image of my learning platform with all my timetables for ONLINE lessons? Nope not good enough.

I'm incredibly disappointed with the state of this system in comparison to the streamlined initial application process. Not to mention you cannot upload images directly, you have to fiddle around with the camera, which doesn't work at ALL if you have multiple. The github team needs to fix their outdated documentation, help support docs, and to rework the entire system because it's utterly useless for people with slightly unusual school circumstances.

rant over

(image of error in email)
![image](https://github.com/user-attachments/assets/1540505c-5c67-4700-8e39-a22e985dc0cc)

Please help though, I use the benefits almost every day and this is really annoying.

Thanks in advance

**Comment by chicouskiluiz** [comment]
Same problem, even selecting this option and sending a document that proves that my course is distance learning, it is still rejected and gives the same error in the email. Unfortunately Github support is terrible.

**Comment by tommy141x** [comment]
Same issue here, provided my tuition bill where each class name includes "Online Learning" and it is still rejected.

**Comment by JanderSilv** [comment]
Same issue here

**Comment by tommy141x** [comment]
![image](https://github.com/user-attachments/assets/5f9ccad3-90b2-43a4-952b-71e8b8d6bf59)
Also unable to make a support ticket for the issue either. Unsure what we are supposed to do

**Reply by MoustacheOff** [reply]
shameful of them

**Comment by RastafaJazzz** [comment]
> Same issue here, provided my tuition bill where each class name includes "Online Learning" and it is still rejected.

**Comment by aseyedia** [comment]
Having the same issue.

https://github.com/orgs/community/discussions/136237

**Comment by gfb-47** [comment]
Same issue.

**Comment by kiq-caique** [comment]
Same issue here.

**Comment by aseyedia** [comment]
I had to ridiculously face my laptop camera at my unofficial transcript being displayed on a computer monitor but I eventually got it.

**Reply by kiq-caique** [reply]
How did you get it? What photo did you send?

**Reply by linkinn** [reply]
How did you get it ?
